backend/core/src/auth/authz_policy.csv

```diff
@@ -9,6 +9,7 @@ p, anonymous, user, true, create
 p, admin, asset, true, .*
 
 p, admin, preference, true, .*
+p, user, preference, true, .*
```

Comment:
AFAIK HRD has decided that partners should have the ability to fully edit a listing. Plus the plan is for the `user` role to only be for partners soon (public users will not have any role), so that should harden this access a bit more.

Reply:
Sounds good, thanks for the explanation!